package.json

```diff
@@ -207,6 +207,7 @@
     "uuid": "^9.0.0",
     "vaul": "^0.9.0",
     "viem": "^1.16.6",
+    "wagmi": "1.4.13",
```

Comment:
Yeah we're bringing this back, no reason to reinvent the wheel when reading data in components - wagmi is just a wrapper around react-query and we would write ours, not so well optimized anyway, using viem.

Since wagmi wraps react-query and uses our viem client under the hood, that's literally the same we would write, but better. Nothing crazy huge either: https://bundlephobia.com/package/wagmi@1.4.13

Note: this is the last v1 before wagmi published v2 which has viem v2 as a dependency, meaning we'd also have to bump viem to v2. Initially went with that and this is nothing super bad to tackle, but this PR is probably not the place for it so staying at v1 for now.

Reply:
Actually bumped viem *and* wagmi to latest, tackled all type errors and renames following breaking change. Now bundle is actually twice smaller 🎉